.gitignore

```diff
@@ -1,19 +1,38 @@
-*.elc
-*.a
+# OCaml files
+# see https://ocaml.org/learn/tutorials/filenames.html
 *.cm?*
-*.o
 *.opt
-*.omc
-*.run
-*~
+*.a
+*.o
+# Deprecated?
+*.annot
+
+# Ocamlbuild (used internally by Dune)
+_build/
+*.byte
+*.native
+
+# Merlin configuring files
+.merlin
+
+# Dune generated files
+*.install
+
+# Beluga debug output files
+debug.out
+
+# ELisp files
+*.elc
+
+# Mac (OS) specific files
 .DS_Store
+
+# Emacs editor setting files
+.dir-locals.el
+*~
+.projectile
+
+# ???
 .build
-.omake*
 release
```

Comment:
@tsani Are these still in use? I mean, `.build` and `release`.

Reply:
AFAIK `.build` is unused, but the RELEASE script creates and uses the `release` directory.

Reply:
@tsani I'm not sure but in my case, RELEASE script does not create one... maybe those errors are something to do with the directory creation.